napari_allencell_annotator/view/images_view.py

```diff
@@ -340,6 +341,9 @@ def remove_image(self, item: FileItem) -> None:
         # TODO when we delete from the model, connect file widget so that it deletes that entry itself without
         # us explicitly calling remove_item on it
         if item.file_path in self._annotator_model.get_all_images():
+            if self.file_widget.currentItem() == item:
```

Comment:
when would this condition trigger?

Reply:
I removed self.file_widget.setCurrentItem(None) in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/7883343d4a7670632f4f084294518823a35afd34. It's not required anymore